code/COVID19 US county Map-Daily Update.ipynb

```diff
@@ -45,15 +48,26 @@
     "tdst=df_Counties_confirmed.columns[-1]\n",
     "print (tdst)\n",
     "tdst1=df_Counties_deaths.columns[-1]\n",
-    "print (tdst1)"
+    "print (tdst1)\n",
+    "\n",
+    "#Change date on yesterday's csv and zipped shapefile in TEST folder\n",
+    "oldZip = covid19RootExport.expanduser() / 'USCounties_JHUmap.zip'\n",
```

Comment:
See comment above - if we're going to use home directories, let's do it consistantly, however I would opt to get out of the 'home directory' context, and just use a root path, as it's not for sure we're coming out of the home directory.

Reply:
I started out only using the UNIX pathing, but some of the path references were giving me an error that it didn't exist, even though checks with the terminal confirmed that they did. I tested out if the error still arose if I wrote out the full path instead of the `~/` shortcut  and it worked. So to avoid using my specific home directory, I used the expanduser() method wherever it failed. 

I'm happy to use a root path instead, just not exactly sure how to implement it. Would we still use `~/` or something more like `../../../`?